packages/manager/modules/pci/src/projects/project/instances/instances.routing.js

```diff
@@ -3,6 +3,7 @@ import find from 'lodash/find';
 import get from 'lodash/get';
```

Comment:
the `catalog` attribute is passed as a parameter to the child component `instance>edit` so I'll keep this one.
I rolledback the code related to the initialization of the `flavor.tags` since we'll be using the `flavor.planCodes.monthly` instead